ush/mesoscale/evs_check_sref_files.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/ksh
 
-set +x
+set -x
 #For testing: 
 #vday=20221003
 #COMINsref=/lfs/h1/ops/prod/com/sref/v7.1
```

Comment:
@RoshanShrestha-NOAA Can this commented out line (#COMINsref=/lfs/h1/ops/prod/com/sref/v7.1) and the commented out lines around it be removed? Thanks!

Reply:
Hi Alicia, Binbin is making changes to sref scripts. If I make those changes, then there will be conflicts later on. Shall I change the sref files too?

Reply:
No problem! Sorry about that!